src/appBootstrapper.tsx

```diff
@@ -235,6 +237,28 @@ if (__DEBUG__ && module.hot) {
     module.hot.accept('./routes', () => render());
 }
 
+async function loadCustomJs() {
+    const downloading: Promise<any>[] = [];
+    const customJsFiles = getServerConfig().custom_js_files;
+    if (!customJsFiles) {
+        return Promise.resolve();
+    }
+    customJsFiles.forEach((customJsFile: CustomJsConfig) => {
+        const p = new Promise(function(resolve, reject) {
+            load(customJsFile.path, (err: any) => {
+                if (err) {
+                    reject(err);
```

Comment:
we should decide how we want to handle failure.  right now i believe that Promise.all below will never resolve if any one of its items fails (check that).  Possible the best thing is to is show some error alert or something.

Reply:
fixed in [fcf585b](https://github.com/cBioPortal/cbioportal-frontend/pull/4677/commits/fcf585b6346b245833c3687ddb4c0d6e216866a5)